CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to cyclops
 
-Thanks for your interest in contributing to cyclops!
+Thanks for your interest in contributing to the cyclops!
```

Comment:
Really, eh? 😂

Reply:
ahh fuck, was actually creating a template. you will get it in the next PR